client/components/common/tce-core/QuestionContainer/index.vue

```diff
@@ -147,6 +147,16 @@ export default {
       this.$set(element.data, 'feedback', { ...element.data.feedback, ...data });
     }
   },
+  watch: {
+    element: {
+      handler() {
+        this.isEditing = !this.element.id;
+        this.editedElement = cloneDeep(this.element);
```

Comment:
This overrides the current state upon BE response (which is a problem if there is a delay).